05_syntax_lab/01.py

```diff
@@ -1,5 +1,28 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+#!/usr/bin/env python
+
+__author__ = 'Eabel'
+
+import sys
+
+def Exercise1():
+    print "Please enter 10 numbers:"
+    max_val = -sys.maxint - 1 # Min int value
+    for i in range(10):
+        try:
+            max_val = max(int(raw_input()),max_val)
+        except ValueError:
+            print "Please enter only numbers"
+            sys.exit()  

```

Comment:
Try the following challenge: When a user types in a non-numeric value, have them type another one up to a total of 10 input numbers read